SMLHelper/Patchers/IngameMenuPatcher.cs

```diff
@@ -7,21 +7,32 @@
     internal class IngameMenuPatcher
     {
         private static readonly List<Action> oneTimeUseOnSaveEvents = new List<Action>();
+        private static readonly List<Action> oneTimeUseOnQuitEvents = new List<Action>();
 
         internal static Action OnSaveEvents;
+        internal static Action OnQuitEvents;
 
         public static void Patch(HarmonyInstance harmony)
         {
             harmony.Patch(AccessTools.Method(typeof(IngameMenu), nameof(IngameMenu.SaveGame)),
-                postfix: new HarmonyMethod(AccessTools.Method(typeof(IngameMenuPatcher), nameof(InvokeEvents))));
+                postfix: new HarmonyMethod(AccessTools.Method(typeof(IngameMenuPatcher), nameof(InvokeSaveEvents))));
+            harmony.Patch(AccessTools.Method(typeof(IngameMenu), nameof(IngameMenu.QuitGame)),
+                postfix: new HarmonyMethod(AccessTools.Method(typeof(IngameMenuPatcher), nameof(InvokeQuitEvents))));
+            harmony.Patch(AccessTools.Method(typeof(IngameMenu), nameof(IngameMenu.QuitGameAsync)),
+                postfix: new HarmonyMethod(AccessTools.Method(typeof(IngameMenuPatcher), nameof(InvokeQuitEvents))));
```

Comment:
`InvokeQuitEvents` gets invoked for both `QuitGame` and `QuitGameAsync`?

Is it unreliable to patch only one of these?

Reply:
Might be on on just quit

Reply:
Just patches quit now.